src/hooks/useIsHomepage.ts

```diff
@@ -0,0 +1,36 @@
+import { usePathname } from 'next/navigation';
+import {
+  JUMPER_BUY_PATH,
+  JUMPER_EXCHANGE_PATH,
+  JUMPER_GAS_PATH,
+  JUMPER_REFUEL_PATH,
+  JUMPER_SWAP_PATH,
+} from 'src/const/urls';
+import { locales } from 'src/i18n';
+
+const checkLocaleIsPath = (path: string | null) => {
+  if (path) {
+    return locales.some(
```

Comment:
What is the purpose of that?

Reply:
In order to figure out which pages do not allow custom theming, I added this hook "useIsDapp". Now, I refactored it -> If not /learn or /loyalty it´s seens as "dapp" and enables theming.